ios/ZulipMobile/ZLPConstants.swift

```diff
@@ -3,15 +3,16 @@ import UIKit
 
 @objc(ZLPConstants)
 class ZLPConstants: NSObject {
-
-
   // For why we include this, see
   //   https://reactnative.dev/docs/0.68/native-modules-ios#exporting-constants
   @objc
   static func requiresMainQueueSetup() -> Bool {
-    // Initialization may be done on any thread; we don't need access to
-    // UIKit.
-    return false
+    // UIApplication, which we use in `constantsToExport`, is provided by
+    // UIKit. I think that means we should return `true` here. From the doc
+    // linked above:
+    // > If your module does not require access to UIKit, then you should
+    // > respond to `+ requiresMainQueueSetup` with NO.
+    return true
   }
```

Comment:
Er it occurs to me that if we want this change, then we probably also want it in `ZLPNotifications` for the same reason.

Reply:
Ah OK, right! Yeah, I think we can remove both things (`constantsToExport` and `requiresMainQueueSetup`). If the module ever wants to export any constants, it can add them back.